admin/settingstree.php

```diff
@@ -0,0 +1,45 @@
+<?php
+// must be run within Dokuwiki
+if(!defined('DOKU_INC')) die();
+
+/**
+ * All DokuWiki plugins to extend the admin function
+ * need to inherit from this class
+ */
+class admin_plugin_dw2pdf_settingstree extends DokuWiki_Admin_Plugin {
+	private $helper = null;
```

Comment:
My impression of your current setup is that it created for each plugin an entry at the admin overview page, which let you open another settingstree config manager page.

My suggestion is to create one settingstree config manager page, but let the user at that page select the plugin that they would manage.

I think that having separate settingtree conf managers will confuse the user, bundling them to one settingtree conf manager let them understand that these are quite similar.

I don't recommend combining Confmanager Plugin and Settingstree Plugin, because Confmanager handles only configuration files like sheme/smileys/abbreviations/mimetypes/etc. Your Settingstree plugin is about more finegrained plugin settings. (Confmanager Plugin is just an example to show you an idea for switching between the plugins)